Core/PrivacyConfiguration.swift

```diff
@@ -76,6 +76,7 @@ public enum PrivacyFeature: String {
     case fingerprintingBattery
     case fingerprintingScreenSize
     case gpc
+    case https
```

Comment:
Maybe `httpsUpgrade` to be more explicit?

Reply:
This matches the feature in the config. But I can change the case name and set a raw string if its too unclear.

Reply:
Yes, please - I'd prefer to be explicit.